org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -68,7 +68,7 @@ define([
          */
         uploader: function (customSettings) {
           var uploader, oldUploadAll;
-          var deferred = $q.defer();
+          var deferred;
```

Comment:
you don't need to define `deferred` here, since you're using it only in the closure
```js
return function (additionalFormData) {
  var deferred = $q.defer();
```

Reply:
It is used outside the uploadAll function, 
https://github.com/civicrm/civihr/blob/aa765c04777f5b992c93f0c7dbfe2f41d551f315/org.civicrm.reqangular/src/common/services/file-upload.js#L84-L84
https://github.com/civicrm/civihr/blob/aa765c04777f5b992c93f0c7dbfe2f41d551f315/org.civicrm.reqangular/src/common/services/file-upload.js#L87